site/data/refDataApi.ts

```diff
@@ -16,7 +16,7 @@ interface FetchStopsInput {
     searchString?: string;
 }
 
-export const fetchStops = async (input: FetchStopsInput) => {
+export const fetchStops = async (input: FetchStopsInput, vehicleMode?: string) => {
```

Comment:
vehicleMode could be of type VehicleMode

Reply:
done